app/views/users/show.html.erb

```diff
@@ -1,2 +1,80 @@
-<h1><%= @user.name %></h1>
-<!-- Other user details -->
+<!DOCTYPE html>
+<html>
+<head>
+  <title><%= @user.name %></title>
+  <style>
+    h1 {
+      color: #333;
+      font-size: 32px;
+      margin-bottom: 20px;
+    }
+
+    p {
+      color: #666;
+      font-size: 18px;
+      margin-bottom: 10px;

```

Comment:
- [ ] Please kindly remove this CSS code that you newly added to your work after the last review, this will make your codebase to be neat. 👍 Placing CSS in a separate file allows developers to manage styles more efficiently.

Reply:
@OyePriscilla  you want me to make a separate CSS file,instead of adding it here in HTML?